src/System.Net.Http/tests/UnitTests/Headers/CurlResponseParserTest.cs

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http;
+
+using Xunit;
+
+namespace System.Net.Http.Tests
+{
+    public class CurlResponseParserTest
+    {
```

Comment:
This test class should be named consistently against the product class it is testing.

The product class is `CurlResponseParseUtils`.  So, this test class should be named `CurlResponseParseUtilsTest' and the filename should be `CurlResponseParseUtilsTest.cs`